tests/manage/z_cluster/test_rook_ceph_operator_log_type.py

```diff
@@ -0,0 +1,172 @@
+from datetime import datetime
+import logging
+import re
+import pytest
+
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check
+from ocs_ci.ocs.resources.pod import (
+    get_pod_logs,
+    get_osd_pods,
+    get_ceph_tools_pod,
+    get_operator_pods,
+)
+from ocs_ci.helpers.helpers import set_configmap_log_level_rook_ceph_operator
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    skipif_ocs_version,
+    bugzilla,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("1962821")
+@skipif_ocs_version("<4.8")
+@pytest.mark.polarion_id("OCS-2581")
+class TestRookCephOperatorLogType(ManageTest):
+    """
+    Test Process:
+    1.Set ROOK_LOG_LEVEL param to "DEBUG"
+    2.Respin OSD pod
+    3.Verify logs contain the expected strings
+    4.Verify logs do not contain the unexpected strings
+    5.Set ROOK_LOG_LEVEL param to "INFO"
+    6.Respin OSD pod
+    7.Verify logs contain the expected strings
+    8.Verify logs do not contain the unexpected strings
+
+    Comment:
+    On INFO mode,  expected log type [I],[E]
+    On DEBUG mode, expected log type [I],[D],[E]
+
+    """
+
+    def teardown(self):
+        set_configmap_log_level_rook_ceph_operator(value="INFO")
+        tool_pod = get_ceph_tools_pod()
+        tool_pod.exec_ceph_cmd(ceph_cmd="ceph crash archive-all", format=None)
+        logging.info(
+            "Perform Ceph and cluster health checks after silencing the ceph warnings"
+        )
```

Comment:
Any reason why we are running ceph crash archive-all?

Reply:
I saw that after I deleted the osd pod the ceph status moved to WARN state

Reply:
done